### PR TITLE
Some small fixes to Batteries

### PR DESCRIPTION
**Pull request by AHaumer**
I had to fix a few typos as well broken links (didn't notice them during the last change regarding the parameters). Additionally I created a (sub-)icon indicating the usage of transient RC elements.

### CODE REVIEW

File: Modelica/Electrical/Batteries/BaseClasses/BaseCellStack.mo [hunk 1]
```diff
@@ -48,16 +48,11 @@ partial model BaseCellStack
...
-    "Specify OCV(SOC) table with SOCmax <= 1");
-  assert(cellData.OCV_SOC[1, 2] >= 0,
-    "Specify OCV(SOC)/OCVmax table with  OCVmin/OCVmax >= 0");
-  assert(cellData.OCV_SOC[end, 2] >= 1,
-    "Specify OCV(SOC)/OCVmax table with max.OCV/OCVmax <= 1");
+  assert(cellData.SOCmax > cellData.SOCmin, "Specify 0 <= SOCmin < SOCmax <= 1");
+  assert(cellData.OCV_SOC[1, 1] >= 0, "Specify OCV(SOC) table with SOCmin >= 0");
+  assert(cellData.OCV_SOC[end, 1] <= 1,  "Specify OCV(SOC) table with SOCmax <= 1");
+  assert(cellData.OCV_SOC[1, 2] >= 0, "Specify OCV(SOC)/OCVmax table with  OCVmin/OCVmax >= 0");
+  assert(cellData.OCV_SOC[end, 2] >= 1, "Specify OCV(SOC)/OCVmax table with max.OCV/OCVmax <= 1");
```
Comment (christiankral) [inline Modelica/Electrical/Batteries/BaseClasses/BaseCellStack.mo]:
```suggestion
  assert(cellData.OCV_SOC[1, 2] >= 0, "Specify OCV(SOC)/OCVmax table with OCVmin/OCVmax >= 0");
  assert(cellData.OCV_SOC[end, 2] >= 1, "Specify OCV(SOC)/OCVmax table with maximum OCV/OCVmax <= 1");
```

Comment (AHaumer) [reply Modelica/Electrical/Batteries/BaseClasses/BaseCellStack.mo]: Hmmm... in my humble opinion there's another bug (I don't understand your suggestion):
  assert(cellData.OCV_SOC[1, 1] >= 0, "Specify OCV(SOC) table with min.SOC >= 0");
  assert(cellData.OCV_SOC[end, 1] <= 1, "Specify OCV(SOC) table with max.SOC <= 1");
  assert(cellData.OCV_SOC[1, 2] >= 0, "Specify OCV(SOC) table with  min.OCV/OCVmax >= 0");
  assert(cellData.OCV_SOC[end, 2] **<=** 1, "Specify OCV(SOC) table with max.OCV/OCVmax <= 1");
See my commit d4cdbe480..46349a27b

**Comment by AHaumer** [comment]
I've also changed sligthly the parameterizaton of the array of RC-elements (the user has to specify the number of RC-elements) to avoid later problems (when using stacks built from cells with different parameters, i.e. a matrix of Ns series connected x Np parallel connected cells (which I'm about to prepare).

**Comment by AHaumer** [comment]
Dear all (especially @christiankral @dietmarw @beutlich ), just a question:
I've implemented an extension of the batteries library (in a students project) with stacks built as a matrix of Ns series connected x Np parallel connected cells (with different parameters), equipped with sensors, to enable the development of a battery management system.
Shall a open a separate PR after this PR has been merged, or shall I implement them in this PR and change the title to "Fixes and extendsions of the Battery Library"?
Have a Merry X-Mas!

**Comment by beutlich** [comment]
@christiankral @dietmarw @DrMartinKuhn Can you please review. Thank you.

**Comment by dietmarw** [comment]
Sorry but I will wait another week or two so that the dust finally settles. I don't like to redo reviews on non-finished PRs.

**Comment by beutlich** [comment]
This PR is rather calm. You probably mixed it up with noisy PR #3289.

**Comment by AHaumer** [comment]
???
Last commit in #3285: 15 days ago
Last commit in #3289: 14 days ago

**Comment by AHaumer** [comment]
@dietmarw 
I kept the extensions out of this PR, and both PRs seem to be rather quit since 2 weeks ...

**Comment by dietmarw** [comment]
@AHaumer Yes I saw that. So just waiting for @christiankral or @DrMartinKuhn to approve so we can merge this one and move on to the extensions.

**Comment by beutlich** [comment]
> 
> 
> Consider [AHaumer#7](https://github.com/AHaumer/ModelicaStandardLibrary/pull/7) to improve readability

Done.

**Pull request merged**